cpp/mcap/include/mcap/writer.hpp

```diff
@@ -388,6 +388,17 @@ class MCAP_PUBLIC McapWriter final {
    */
   Status write(const Metadata& metadata);
 
+  /**
+   * @brief Set the chunk size, compression type, and compression level to use
+   * for the next Chunk record. This will close any open chunk and start a new
```

Comment:
I find it strange to have a function called `setChunkOptions` that has this side effect.

Reply:
What about removing the side effect? Set these options on the next chunk that is started? The user can close the current chunk if they'd like?

Your documentation already has the language "for the next Chunk record" so all you'd change is removing "This will close any open chunk".

Is there a method to close the current chunk?

Reply:
Yes there is a `closeLastChunk()` method. I will update this to use additional McapWriter member variables for "next chunk settings".